nextmind-stretch-server/app.py

```diff
@@ -4,42 +4,84 @@
 from starlette.responses import JSONResponse
 from starlette.routing import Route
 
-path_to_commands = "/home/hello-robot/Chatbot/moveGrip.py"
+path_to_commands = "/home/hello-robot/Chatbot"
 
 
 async def open_grip(request):  # scan:ignore
     move_amount = request.path_params["amount"]
-    os.system("python " + path_to_commands + " 'open' " + str(move_amount))
+    os.system("python " + path_to_commands + "/moveGrip.py 'open' " + str(move_amount))
```

Comment:
This is what I was saying should be a function - I didn't mean truncate path_to_commands

Reply:
changed to run_command("moveGrip",move_amount), for all commands

Reply:
done